test/error_handling/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+
+macro(run_test case)
+get_filename_component(test_name ${case} NAME_WE)
+set(test_comment "! ")
+add_test(NAME ${test_name}.error_handling
```

Comment:
maybe make the suffix here just `.Error`? note the consistent suffix case convention (though maybe they should all be lower case?).